test/e2e/provider/00-assert.yaml

```diff
@@ -18,7 +18,7 @@ spec:
     services:
       sos:
         buckets:
-          - bucket.test.1
+          - bucket.test.e2e
```

Comment:
Out of curiousity: Why do we delimit bucket names with `.` instead of `-` as we do in cloudscale?

Reply:
I had some issues during development with "-". I thought it's not allowed in exoscale. I did some manual tests now and everything seem to work as expected. I will change from `.` to `-`.